pmlc/dialect/pxa/transforms/passes.td

```diff
@@ -8,6 +8,11 @@ def AutoTileExample : FunctionPass<"pxa-autotile-example"> {
   let constructor = "pmlc::dialect::pxa::createAutoTileExamplePass()";
 }
 
+def VectorizeExample : FunctionPass<"pxa-vectorize-example"> {
+  let summary = "Vectorizes woith vector register of 8 elements";
```

Comment:
typo: with

Reply:
Is this specific to 8 elements? Can we make this configurable?

Reply:
Yes as per above.